packages/ppx_inline_test/ppx_inline_test.0.13.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/ppx_inline_test"
+bug-reports: "https://github.com/janestreet/ppx_inline_test/issues"
+dev-repo: "git+https://github.com/janestreet/ppx_inline_test.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/ppx_inline_test/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"  {>= "4.04.2"}
+  "base"   {>= "v0.13" & < "v0.14"}
+  "dune"   {>= "1.5.1"}
```

Comment:
According to the `dune-project` file this should be 1.10
```suggestion
  "dune"   {>= "1.10"}
```

Reply:
Thanks! I missed this